ci/kind/test-clickhouse-migrate.sh

```diff
@@ -0,0 +1,262 @@
+                "projects.registry.vmware.com/antrea/clickhouse-operator:$CH_OPERATOR_FROM_TAG" \
+                "projects.registry.vmware.com/antrea/metrics-exporter:$CH_OPERATOR_FROM_TAG" \
+                "projects.registry.vmware.com/antrea/clickhouse-operator:0.21.0" \
+                "projects.registry.vmware.com/antrea/metrics-exporter:0.21.0" \
+                "projects.registry.vmware.com/antrea/theia-zookeeper:3.8.0" \
+                "projects.registry.vmware.com/antrea/theia-grafana:8.3.3" \
+                "projects.registry.vmware.com/antrea/antrea-ubuntu:$ANTREA_FROM_TAG" \
+                "projects.registry.vmware.com/antrea/theia-clickhouse-monitor:$THEIA_FROM_TAG" \
+                "projects.registry.vmware.com/antrea/theia-clickhouse-server:$CLICKHOUSE_FROM_TAG" \
+                "antrea/antrea-ubuntu:latest")
+
+for img in "${DOCKER_IMAGES[@]}"; do
+    echo "Pulling $img"
+    for i in `seq 3`; do
+        docker pull $img > /dev/null && break
+        sleep 1
+    done
+done
+
+DOCKER_IMAGES+=("projects.registry.vmware.com/antrea/theia-clickhouse-monitor:latest\
+                 projects.registry.vmware.com/antrea/theia-clickhouse-server:latest")
+
+echo "Creating Kind cluster"
+IMAGES="${DOCKER_IMAGES[@]}"
+$THIS_DIR/kind-setup.sh create kind --images "$IMAGES"
+
+# When running this script as part of a Github Action, we do *not* want to use
+# the pre-installed version of kustomize, as it is a snap and cannot access
+# /tmp. See:
+#  * https://github.com/actions/virtual-environments/issues/1514
+#  * https://forum.snapcraft.io/t/interfaces-allow-access-tmp-directory/5129
+unset KUSTOMIZE
+
+# Load latest Antrea yaml files
+TMP_DIR=$(mktemp -d $(dirname $0)/tmp.XXXXXXXX)
+curl -o $TMP_DIR/antrea.yml https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea.yml
+
+sed -i -e "s|image: \"projects.registry.vmware.com/antrea/antrea-ubuntu:latest\"|image: \"antrea/antrea-ubuntu:latest\"|g" $TMP_DIR/antrea.yml
+sed -i -e "s|type: RollingUpdate|type: OnDelete|g" $TMP_DIR/antrea.yml
+# Load latest Theia yaml files
+docker exec -i kind-control-plane dd of=/root/antrea-new.yml < $TMP_DIR/antrea.yml
+$ROOT_DIR/hack/generate-manifest.sh --local /data/clickhouse --no-grafana | docker exec -i kind-control-plane dd of=/root/flow-visibility-new.yml
+docker exec -i kind-control-plane dd of=/root/clickhouse-operator-install-bundle-new.yaml < $ROOT_DIR/build/charts/theia/crds/clickhouse-operator-install-bundle.yaml
+rm -rf $TMP_DIR
+
+# Load previous version yaml files from Antrea repo
+TMP_DIR=$(mktemp -d $(dirname $0)/tmp.XXXXXXXX)
+git clone --branch $ANTREA_FROM_TAG --depth 1 https://github.com/antrea-io/antrea.git $TMP_DIR
+
+pushd $TMP_DIR > /dev/null
+export IMG_NAME=projects.registry.vmware.com/antrea/antrea-ubuntu
+export IMG_TAG=$ANTREA_FROM_TAG
+./hack/generate-manifest.sh --mode release | docker exec -i kind-control-plane dd of=/root/antrea.yml
+popd
+rm -rf $TMP_DIR
```

Comment:
Since we plan to use the same Antrea yaml file for the ClickHouse migration test, maybe can remove all the contents related to ANTREA_FROM_TAG in this file?

Reply:
We will require the ANTREA_FROM_TAG for installing the last release version of Theia for checking the upgrade from a release to the latest code. Similar arguments hold for THEIA_FROM_TAG ymls